src/playbooks/with_build/vsp_preupgrade_health.yml

```diff
@@ -53,3 +53,4 @@
     primary_vsds_report_name: vstat_preupgrade_health_report_for_vsd.txt
     vscs_vstat_report_name: vstat_preupgrade_health_report_for_vsc.txt
     vrss_report_name: vstat_preupgrade_health_report_for_vrs.txt
+    portals_report_name: vsp_preupgrade_health_report_for_sdwan_portal
```

Comment:
same comment here

Reply:
the .txt extension gets added during portal health when setting the report path so I didn't include it when setting the report name